### DISCUSSION TITLE
Configuration Issue

### DISCUSSION THREAD

**Discussion by N7Pankake** [jeffreylanters/react-unity-webgl#299] [Help Wanted!] [answered]
Hi I was having the same issue than this person [Configuration Issue #131](https://github.com/jeffreylanters/react-unity-webgl/issues/131#) 

The legacy documentation no longer exist it says 404 error, also I'm having the same issue and I can't understand why it doesn't work because I have it on my public folder.

This is how I have my folder structure

![Unity set](https://user-images.githubusercontent.com/8898953/148276526-2fcb5aa6-768e-4b2e-ac97-84639ccd3be5.png)

And this is what I have in my JS file where I'm trying to render the Unity project

```
import React from 'react'
import Unity, { UnityContent } from "react-unity-webgl";

let unityContent = new UnityContent(
  "Megaman/Build/Megaman.json",
  "Megaman/Build/UnityLoader.js"
);

function MegamanGame() {
    return (
        <div><Unity unityContent={unityContent} /></div>
      
    )
}

export default MegamanGame
```

**Comment by jeffreylanters** [comment, marked as answer]
Hi. The legacy documentation moved to the readme under the 7.x tag. You should be able to find it there. 

Looking at your screenshot, your structure and React code looks just fine. Are you getting any errors in the console?

**Reply by jeffreylanters** [reply]
Hm, I notice that your MegamanGame component is place into a sub directory. Is this view served using a nested path? For example something like: "https://example.com/games/megaman"?

Because if that's the case, your build might not be found by the module because your you've entered relative URLs. This way the module would try to load the files from in the same example: "https://example.com/games/Megaman/Build/Megaman.json".

Could you try using absolute URLs by simply putting a / in front of every URL?

**Reply by N7Pankake** [reply]
Yes is being used as a nested path let me try that.

**Reply by N7Pankake** [reply]
So for example if the URL is:

<Route path = "/" element={<AppToolBar/>} >
            <Route path="/MegamanClone/Game" element={( <> <MegamanGame /> <Footer /> </> )} >
            </Route>
</Route>

Should I just set it up as /MegamanClone/Game/Build/Megaman ?

**Reply by jeffreylanters** [reply]
Could you try the following:

```ja
import React from 'react'
import Unity, { UnityContent } from "react-unity-webgl";

let unityContent = new UnityContent(
  "/Megaman/Build/Megaman.json",
  "/Megaman/Build/UnityLoader.js"
);

function MegamanGame() {
    return (
        <div><Unity unityContent={unityContent} /></div>
      
    )
}

export default MegamanGame

```

**Reply by N7Pankake** [reply]
Oh wow I did try that before but it didn't work and now it does well not gonna complain, thanks for the help.

Edit: Okay just notice all I was missing is the / I see
